Source/Build Specs/Measurement Plug-In SDK Service.vipb

```diff
@@ -136,27 +136,21 @@
         <Additional_Destination_Index>1</Additional_Destination_Index>
       </Destination_Overrides>
       <Destination_Overrides>
-        <Path>Runtime/Palette Files/Measurement</Path>
-        <Destination>0</Destination>
-        <Additional_Destination>true</Additional_Destination>
-        <Additional_Destination_Index>0</Additional_Destination_Index>
-      </Destination_Overrides>
-      <Destination_Overrides>
-        <Path>Runtime/Palette Files</Path>
+        <Path>Runtime/Clients</Path>
         <Destination>0</Destination>
         <Additional_Destination>false</Additional_Destination>
         <Additional_Destination_Index>0</Additional_Destination_Index>
       </Destination_Overrides>
       <Destination_Overrides>
-        <Path>Runtime/Clients</Path>
+        <Path>Runtime/TestStand</Path>
         <Destination>0</Destination>
         <Additional_Destination>false</Additional_Destination>
         <Additional_Destination_Index>0</Additional_Destination_Index>
       </Destination_Overrides>
       <Destination_Overrides>
-        <Path>Runtime/TestStand</Path>
+        <Path>Palette Files/Measurement</Path>
```

Comment:
With this change, you now have two packages which are installing `Measurement Plug_In SDK.mnu`. I can't necessarily think of anything that will break because of this, but it is weird. I think it would be better if each palette file were owned by exactly one package.